lib/service/modules/tempObjectStore.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2016 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+'use strict';
+
+module.exports = (persistentTempObjectStore) => {
```

Comment:
I reneged on this and decided to use ES6 default params. The DLC-CLI should (and likely will) be upgraded to support node 6 sooner rather than later